pkg/reconcilermanager/controllers/reposync_controller_test.go

```diff
@@ -3710,6 +3810,11 @@ func validateDeployments(wants map[core.ID]*appsv1.Deployment, fakeDynamicClient
 					if diff := cmp.Diff(i.Resources, j.Resources); diff != "" {
 						return errors.Errorf("Unexpected resources found for the %q container of %q, diff %s", i.Name, id, diff)
 					}
+
+					// Compare Args
+					if diff := cmp.Diff(i.Args, j.Args); diff != "" {
```

Comment:
nit: this could use testutils.AssertEqual, which prints a diff for you automatically.

Reply:
it looks like using `testutil.AssertEqual` will require refactoring `validateDeployment` func to take `t *testing.T` as param, which might be worth a separate refactoring PR to change all `cmp.Diff` usage within `validateDeployment`. Unless you think it's really important to include this refactor in this PR, i think it'll be better to be done in separate changes. Thoughts?

Reply:
not a big deal.